codemeta.json

```diff
@@ -369,6 +369,6 @@
         "click>=7.0",
         "pydantic>=2.5.0",
         "iminuit>=2.8.0",
-        "matplotlib>=3.4"
+        "matplotlib>=3.4, <3.10"
```

Comment:
Why this change?

Reply:
This is a change introduced by the codemeta bot. It reflects modifications in setup.cfg  from #5659